gcloud-java-bigquery/src/main/java/com/google/gcloud/spi/DefaultBigQueryRpc.java

```diff
@@ -91,7 +92,14 @@ private static BigQueryException translate(IOException exception) {
 
   private static BigQueryException translate(GoogleJsonError exception) {
     boolean retryable = RETRYABLE_CODES.contains(exception.getCode());
-    return new BigQueryException(exception.getCode(), exception.getMessage(), retryable);
+    BigQueryError bigqueryError = null;
+    if (exception.getErrors() != null  && !exception.getErrors().isEmpty()) {
+      GoogleJsonError.ErrorInfo error = exception.getErrors().get(0);
```

Comment:
And if I am not mistaken this is also the case in Datastore.
But no, I don't know whether there are some cases in which the `errors` array contains more than one value. Maybe @aozarov has more information than I do?